docs/multi-organization-guide.md

```diff
@@ -0,0 +1,182 @@
+
+Implementing such an approach is out of scope of this guide and it heavily depends on the specific source code & CICD tool in use. 
+
+
+## Best Practices for Multi-Organization ADF setups
+If you want to run ADF in a multi-organization setup, there are various best practices that should be followed.
+When following these recommendations, the content of the repository `aws-deployment-framework-bootstrap` and `aws-deployment-framework-pipelines` should be stage agnostic. 
+This means that you can copy and paste the content of any of those two repositories into any AWS Organization stage ("dev", "int", "prod") and ADF will behave exactly 
+
+### 1. Create a dedicated adfconfig.yml Per AWS Organization
+One challenge with synchronising the aws-deployment-framework-bootstrap repository
+across AWS Organizations is that the contents of the `adfconfig.yml` configuration
+file is typically tailored to the ADF installation. The can be solved by adding a 
+custom adfconfig file for the given organization.
+
+Adding a configuration file with the name pattern `adfconfig.{organization id}.yml`
+in the root of the `aws-deployment-framework-bootstrap` repository will take
+precedence over the default `adfconfig.yml` settings file for that organization.
+
+For each AWS organization used with the ADF Framework setup an additional adfconfig
+file can be defined.
+
+
+### 2. Design Multi-Organization ADF Deployment Maps
+The Deployment Maps for ADF exist in the codecommit repository
+`aws-deployment-framwork-pipelines` within the deployment 
+account. Some additional Multi-Org challenges exist when defining targets for Deployments. As a high-level goal, a deployment map should be setup in such a way, that it can be copied over from one ADF instance to another without breaking / requiring any change.
+
+The following considerations should be observed when creating Deployment Maps for an multi-organization ADF setup:
+1. Create Organization-agnostic deployment maps
+    - As a best-practice, deployment maps should be free of any hardocded AWS Account Ids for Deployment Map Targets, unless the deployment is destined for a single AWS Organization.
+    - Instead, target AWS accounts via `Account Names`, `Account Tags` or `OU Paths`
+    which ADF will then use to dynamically generate the respective Account IDs 
+    Target List when updating the Pipelines. 
+2. Consider AWS service limits for AWS CodePipeline
+    - In a large enteprise setup the number of targets in a "prod" AWS Organizations AWS CodePipeline
+  stage may be much greater than in predecessing stages of the "int" and "prod" AWS Organizations.
+    - Review the Codepipeline action limitations.
+    - ADF distributes targets across AWS CodePipeline stages within a AWS CodePipeline on a best efforts basis, however 
+    deployments may need to be distributed across multiple AWS CodePipeline when upper limits are reached.
```

Comment:
```suggestion
    However, deployments may need to be distributed across multiple deployment pipelines if they target hundreds of accounts in an AWS Organization.
```

Reply:
Changed it according to your suggestion. I also slightly reordered the section.